src/Utils/HtmlTable.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace SMW\Utils;
+
+use Html;
+
+/**
+ * @license GNU GPL v2+
+ * @since 3.0
+ *
+ * @author mwjames
+ */
+class HtmlTable {
+
+	/**
+	 * @since 3.0
+	 *
+	 * @param string $html
+	 * @param array $attributes
+	 *
+	 * @return string
+	 */
+	public static function table( $html = '', array $attributes = array() ) {
```

Comment:
Clearly one of those cases where using static is not a total disaster, but still. Esp since you delegate to MediaWiki's Html... I'd be using an instance